modules/mesh-task/variables.tf

```diff
@@ -87,3 +89,15 @@ variable "tags" {
   type        = map(string)
   default     = {}
 }
+
+variable "tls" {
+  description = "Whether to enable TLS for the mesh-task."
+  type        = bool
+  default     = false
+}
+
+variable "consul_server_ca_cert_arn" {
+  description = "The ARN of the secret of the Consul server CA certificate."
```

Comment:
Should we add something to `validation.tf` that this is set if `tls` is true?

Reply:
Added!